### DISCUSSION TITLE
Add autocomplete to search

### DISCUSSION THREAD

**Discussion by kaysond** [linuxserver/Heimdall#727] [Ideas]
It would be great if the search bar had autocomplete as an option!

**Comment by KodeStar** [comment]
Not sure that makes any sense? In order to autocomplete something it needs something to reference, you want me to add a dictionary?

**Comment by kaysond** [comment]
I'm talking about the something like the autocomplete that you get if you use the search engine's own search page.

For example, for google you would do it using google custom search: https://developers.google.com/custom-search/docs/tutorial/autocomplete

Right now you've just got a simple form that passes the query to a page based on whichever search provider is selected in the options. This would require some extra javascript, and in the case of google, the user would have to provide their own custom search engine id. I'm not sure what the API for other search engines looks like.

**Comment by KodeStar** [comment]
Thanks for the additional info.  I'll look into it.

**Comment by vladshved** [comment]
Please make it as an option, I don't like autocomplete by default.
Thanks
